### DISCUSSION TITLE
How to change max target version?

### DISCUSSION THREAD

**Discussion by jaigak** [microsoft/WindowsAppSDK#1084] [Q&A] [answered]
I'm trying to reference a C++ Runtime Component project from a C# WinUI 3 app which targets the latest Insider SDK however I get this error while building the C# app: ```C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2526,6): warning MSB3851: This project targets "Windows, Version=10.0.19041.0", but it is attempting to reference "C:\Users\jaiga\Desktop\source\OneToolkit\x64\Debug\OneToolkit\OneToolkit.winmd" targeting "UAP,Version=10.0.22000.0", which is invalid.```

I checked the packaging project and it is set to 10.0.22000.0 but I can't find anything like that for the app project.

**Comment by jaigak** [comment, marked as answer]
Trying to set the target framework to net5.0-windows10.0.22000.0 doesn't work as it doesn't recognize that framework.

**Reply by manodasanW** [reply]
To target the insider SDK TFM in a .NET 5 app, you need to set the `UseWindowsSdkPreview` property added in the .NET SDK 5.0.8 update.

More details [here](https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewSDK) under Install the SDK Insider Preview .

**Comment by jaigak** [comment]
I'm able to change the max target version in a C++ project but not in a C# project.
